build.sbt

```diff
@@ -29,6 +29,10 @@ def evictionSettings: Seq[Setting[_]] = Seq(
     .withWarnDirectEvictions(false)
 )
 
+def overridesScalaParserCombinators = Seq(
```

Comment:
@ignasi35 decided to use an `dependencyOverrides` since I feel this is safer than exclude an transitive dependency.